src/machine/core.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include "common/logging.h"
 #include "execute/alu.h"
+#include "memory/tlb/tlb.h"
```

Comment:
The TLB integration has to be solved such way that internal core logic does not need know how it works and how it is implemented. Same for tests etc.